transcom/mymove
# B 21449 int

## [B-21449](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1015488)

## Summary

Updates the office app so that when an office user is adding a ppm for a customer, the app will give a descriptive message when DTOD is down.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test
*NOTE* Prior to testing, be sure to turn DTOD_SIMULATE_OUTAGE to true in .envrc

1. Access the office application
2. Login as a Service counselor
3. Select a move from the queue
4. Attempt to add a new PPM to the move
5. Verify that the error message matches the attached screenshot

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-10-24 at 1 13 03 PM](https://github.com/user-attachments/assets/b65670a5-dc77-4cf9-bd75-79c1aee6b2be)
